### PR TITLE
Teet 1226 contract page

### PR DESCRIPTION
**Pull request by jkorpelin**

### CODE REVIEW

File: app/frontend/resources/routes.edn [hunk 1]
```diff
@@ -224,12 +224,16 @@
...
-  :title (tr [:contracts :contracts-listing])}
+  :crumb (tr [:contract :contracts-listing])
+  :title (tr [:contract :contracts-listing])}
 
  :contract
  {:path "/contracts/:contract-ids"                          ;; /contracts/{procurement-id}-{procurement-part-id}
   :view teet.contract.contract-view/contract-page
+  :extenal-links [teet.contract-contract-view/contract-links state]
+  :crumb (teet.contract.contract-model/contract-url-id state)
+  :title (get-in state [:thk.contract/name])
```
Comment (aleksievelizhanin) [inline app/frontend/resources/routes.edn:235]: Can be simple (:thk.contract/name state)

File: app/frontend/src/cljs/teet/contract/contract_menu.cljs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        anchor-el (atom nil)
+        toggle-open! #(do
+                        (swap! open? not)
+                        (.blur @anchor-el))
+        set-anchor! #(reset! anchor-el %)]
+    (common/component
+      (hotkeys/hotkey "Q" toggle-open!)
+      (fn [e! app contract]
+        [:div
+         [buttons/small-button-primary {:start-icon (r/as-element [icons/navigation-menu])
```
Comment (aleksievelizhanin) [inline app/frontend/src/cljs/teet/contract/contract_menu.cljs:59]: Shall we add data-cy="open-contract-button" for elements like buttons to (future) tests?

**Review by aleksievelizhanin — APPROVED** [review]
Looks legit. Some minor comments/questions.

File: app/frontend/src/cljs/teet/contract/contract_menu.cljs (hunk 2 above)
Comment (jkorpelin) [reply app/frontend/src/cljs/teet/contract/contract_menu.cljs:59]: I would personally just add the test attributes when the tests are done.

**Pull request merged**